assets/components/src/color-picker/index.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * WordPress dependencies.
+ */
+import { ColorIndicator, ColorPalette } from '@wordpress/components';
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import './style.scss';
+
+/**
+ * Color Picker.
+ */
+class ColorPicker extends Component {
+	state = {
+		color: '#ffffff',
+	};
+
+	/**
+	 * Render.
+	 */
+	render() {
+		const { hasDefaultColors, label, suggestedColors, ...otherProps } = this.props;
+		const { color } = this.state;
+		const defaultColors = [
+			{
+				name: __( 'navy' ),
```

Comment:
The `name` of the colors don't seem to be displayed anywhere. Are they required and/or should they be displayed?

Reply:
It's for accessibility reason